app/components/Views/confirmations/external/staking/hooks/useStakingDetails.ts

```diff
@@ -9,14 +9,19 @@ import { selectConversionRateByChainId } from '../../../../../../selectors/curre
 import { fromWei, hexToBN } from '../../../../../../util/number';
 import useFiatFormatter from '../../../../../UI/SimulationDetails/FiatDisplay/useFiatFormatter';
 import { useTransactionMetadataRequest } from '../../../hooks/transactions/useTransactionMetadataRequest';
+import { useNetworkInfo } from '../../../../../../selectors/selectedNetworkController';
+import { getDecimalChainId } from '../../../../../../util/networks';
 
-export const useStakingDetails = () => {
+export const useStakingDetails = (chainId?: number) => {
```

Comment:
```suggestion
export const useStakingDetails = () => {
```

Apparently this hook is just used in `app/components/Views/confirmations/external/staking/components/staking-details/staking-details.tsx` and it's not passing any `chainId` at all. My guess is these are unintentionally added?

Also just for the note; as this hook used in the redesigned confirmations - if we need to use `chainId` then we always derive it from `transactionMeta.chainId`. Imported `useNetworkInfo` is pointing globally selected network - this is not something we want to do in confirmations - it's against multi-chain compatible.

Reply:
This has been removed

Reply:
Thank you @nickewansmith